storops/unity/resource/system.py

```diff
@@ -246,11 +246,11 @@ def get_filesystem(self, _id=None, name=None, **filters):
 
     def get_cifs_share(self, _id=None, name=None, **filters):
         return self._get_unity_rsc(UnityCifsShareList, _id=_id, name=name,
-                                   **filters)
+                                   allow_dup=True, **filters)
```

Comment:
In this case, a list is returned, it is different from the original method which a single object is returned, will the storops users such as OpenStack be impacted?

Reply:
First, in the case of local replication, there could be the case of more than two shares are with the same name. If `allow_dup` is not supported, `_get_unity_rsc` will fail when the name is specified, because the original codes don't support more than two shares with the same name.

Second, if it is the case of two shares with the same name, the codes of replication feature will handle these two shares returned very well. If it is the case of only one share with the specific name, `_get_unity_rsc` returns only the single object even `allow_dup=True`, that is, the behavior doesn't change. So, it doesn't impact the existing OpenStack codes.